src/main/kotlin/interpreter/Value.kt

```diff
@@ -0,0 +1,7 @@
+package org.example.interpreter
+
+interface Value {}
+class NumberValue(val value: Double) : Value
+class StringValue(val value: String) : Value
+class NullValue : Value
```

Comment:
We could join NullValue and VoidValue into one. What do you think? Is it more clear to have the return value and the Environment value separate?